appendices/assignments.md

```diff
@@ -0,0 +1,197 @@
+    Pull Request, check if an issue already exists for improving something
+    and comment in the Issue that you will try to address the issue in a Pull
+    Request. Otherwise, open a new Issue and ask if the maintainer would be open
+    to Pull Requests.
+
+## Third project: Each group creating a package
+
+This assignment is *optional* as it depends on multiple factors like class size,
+duration of course, or virtual vs in-person. This would be the third and final 
+deliverable for the learners. There are no final exercises in each chapter that
+learners need to complete related to this assignment. Exercises and the
+structure of the group work would be determined by the instructor.
+This assignment would best be due sometime after the teamwork lesson (REF).
+
+- Using the same process as was done with the first project assignment,
+complete a package as a team, using the tools taught in the team and community
+chapters (TODO: REF).
+
+## Rubric and expectations of the learner
+
+TODO: Can fill this out more as we work on book
+
+The expectations for completing the project assignments change as the course 
+progresses.
+
+- After the first Git lesson (REF), project should have frequent and consistent
+commits and, after the GitHub lesson (REF), pushes to GitHub.
+The GitHub history will be checked to confirm this happens.
+- Tasks in the final chapter exercises should be completed shortly after
+finishing the chapter.
+
+## For instructors (add once file is merged)
+
+Making an interesting project assignment can be quite difficult, so this section
+gives some suggestions and guidance on how what the specific project should be.
+The specific type and number of assignments are of course ultimately decided by
+the instructor of the course, but at the least, this course is designed to with
+two main project assignments in mind, as described above. The specific details of
+the assignments are completed by the instructor, however, this section contains
+some general advice and guidelines.
+
+- The first project package description already provides a suggested package to
+create. As the instructor you can modify this example to focus on weather
+from a specific city, a specific time period, or different aspects of the weather
+(e.g. temperature only).
+- While the project package does not necessarily need to involve data, it might
+be more motivating for the learners to work on if the project did include data of 
+some time. R is a language predominantly used by researchers and data scientists
+for doing data analysis, visualization, and related reporting. Given this,
+learners taking this course are probably already familiar with these tasks and
+would benefit more from the course if they see and use an application they are
+familiar with.
+    - If it doesn't involve data, it's still recommended to structure the project
+    package around using or working with data in some way.
```

Comment:
What does this mean? Doesn't "using or working with data" mean that it DOES involve data still?

Reply:
Hmm, good point. I guess I was thinking e.g. an API that interacts with a website that has data, but don't actually do any work directly on the data, if that makes sense. Given R's focus on data, I don't think we should recommend straying too far from that.